huxaiphaer/flutter-to-do-task
# Implement continuous integration

### What does this PR do ?
This PR implements continuous integration of the application using TravisCI and WeTransfer.

### Description of task to be completed.

- Implement continuous integration.

### How should this be manually tested?

- Check the whether the readMe has a badge. Click on the badge( you will be directed to TravisCI)

### Link to the task.
[Card link](https://github.com/huxaiphaer/flutter-to-do-task/projects/2#card-31236440)
